core/base/supported.go

```diff
@@ -398,12 +398,12 @@ var centosSeries = map[SeriesName]seriesVersion{
 		WorkloadType: OtherWorkloadType,
 		OS:           CentosOS,
 		Version:      "7",
-		Supported:    true,
+		Supported:    false,
 	},
 	Centos9: {
 		WorkloadType: OtherWorkloadType,
 		OS:           CentosOS,
 		Version:      "9",
-		Supported:    true,
+		Supported:    false,
 	},
 }
```

Comment:
If we set this to false, then are we dropping support for centos in 4 completely? If that's the case we should just remove this list.